pkgs/applications/version-management/git-interactive-rebase-tool/default.nix

```
@@ -27,22 +27,24 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs = lib.optionals stdenv.isDarwin [ libiconv Security ];
 
-  checkFlags = [
-    "--skip=external_editor::tests::edit_success"
-    "--skip=external_editor::tests::editor_non_zero_exit"
-    "--skip=external_editor::tests::empty_edit_abort_rebase"
-    "--skip=external_editor::tests::empty_edit_error"
-    "--skip=external_editor::tests::empty_edit_noop"
-    "--skip=external_editor::tests::empty_edit_re_edit_rebase_file"
-    "--skip=external_editor::tests::empty_edit_undo_and_edit"
+  # Compilation during tests fails if this env var is not set.
+  preCheck = "export GIRT_BUILD_GIT_HASH=${version}";
+  postCheck = "unset GIRT_BUILD_GIT_HASH";
```

```suggestion
  preCheck = ''
    export GIRT_BUILD_GIT_HASH=${version}
  '';

```
we don't really need to cleanup up after ourselves in the sandbox

---

Disagree, since this is effectively a hack just for the dev/test build, we should unset it right after the corresponding phase IMHO.

---

fine then